Example/Controllers/SwiftUI/Features/Settings/Features/UniversalBridgeView.swift

```diff
@@ -0,0 +1,41 @@
+import SwiftUI
+import MiniApp
+
+struct UniversalBridgeView: View {
+    @ObservedObject var viewModel: MiniAppSettingsViewModel
+
+    @State var textToSend: String = "{\\\"data\\\":\\\"This is a sample json information from host app.\\\"}"
```

Comment:
I think the string can be escaped with one backslash only 
`"{\"data\":\"This is a sample json information from host app.\"}"`

Reply:
@laubengaier , This is not a json object it is a complete string that will be added to the json payload to be set so those backslash are needed.